.config/bash/environments

```diff
@@ -0,0 +1,28 @@
+FNM_PATH=`which fnm`
+NIX_PROFILE=$HOME/.nix-profile/etc/profile.d/nix.sh
+PYENV_PATH=`which pyenv`
+
+if [[ -x $FNM_PATH ]]; then
+  eval "`fnm env`"
+fi
+
+if [[ -x $PYENV_PATH ]]; then
+  eval "$(pyenv init -)"
+  eval "$(pyenv virtualenv-init -)"
+fi
+
+if [[ -f $NIX_PROFILE ]]; then
+  source $NIX_PROFILE
+fi
+
+export XDG_DATA_HOME="$HOME/.local/share"
+export XDG_CONFIG_HOME="$HOME/.config"
+export XDG_STATE_HOME="$HOME/.local/state"
+export XDG_CACHE_HOME="$HOME/.cache"
+export XDG_RUNTIME_DIR="/run/user/$UID"
+
+export ACKRC="$XDG_CONFIG_HOME/ack/ackrc"
+export PSQLRC="$XDG_CONFIG_HOME/pg/psqlrc"
+export XINITRC="$XDG_CONFIG_HOME/X11/xinitrc"
+
+# vim: set ft=sh:
```

Comment:
perhaps the env var for setting qt theme settings would be useful here

Reply:
What variable is that? and what's the most sensible default?

Reply:
`QT_STYLE_OVERRIDE`
sensible default would be `gtk2`

https://wiki.archlinux.org/title/Qt#Appearance

or 

`QT_QPA_PLATFORMTHEME=qt5ct` but needs qt5ct installed

Reply:
we can set both, the second could be done by:

```bash
QT5CT_PATH=`which qt5ct`

if [[ -x $QT5CT_PATH ]]; then
  export QT_QPA_PLATFORMTHEME=qt5ct
fi
```